src/main/resources/hudson/plugins/throttleconcurrents/ThrottleJobProperty/config.jelly

```diff
@@ -20,6 +20,18 @@
              field="maxConcurrentPerNode">
       <f:textbox />
     </f:entry>
+
+    <f:optionalBlock name="limitOneJobWithMatchingParams"
+                     title="${%Prevent multiple jobs with identical parameters from running concurrently}"
+                     inline="true"
+                     checked="${instance.limitOneJobWithMatchingParams}">
+      <f:entry title="${%List of parameters to check (comma-separated)}"
+               field="paramsToUseForLimit"
+               help="${descriptor.getHelpFile('limitOneJobWithMatchingParams')}">
+        <f:textbox />
```

Comment:
I found the root cause.
You can use `field` instead of `name` and `checked` for `optionalBlock`.
It displays the help for the `optionalBlock` without specifying `help` explicitly.